sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/implementation/models/AcsAdvancedMessageChannelEventError.java

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.messaging.eventgrid.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
+
+/**
+ * Advanced Message Channel Event Error.
+ */
+@Fluent
+public final class AcsAdvancedMessageChannelEventError
```

Comment:
Have we tried using the error type defined in the common-types swagger?https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/data-plane/v1/types.json

If the swagger is updated to use this type, then the generator will use ResponseError.

Reply:
discussed offline, this one is going to stay as is.